go.mod

```diff
@@ -17,13 +17,13 @@ require (
 	github.com/prometheus/common v0.37.0
 	github.com/spf13/cobra v1.6.1
 	google.golang.org/grpc v1.50.1
-	k8s.io/api v0.26.0-rc.0
-	k8s.io/apimachinery v0.26.0-rc.0
-	k8s.io/client-go v0.26.0-rc.0
-	k8s.io/component-base v0.26.0-rc.0
-	k8s.io/component-helpers v0.26.0-rc.0
+	k8s.io/api v0.26.0
+	k8s.io/apimachinery v0.26.1-rc.0
```

Comment:
Any reason why this one is v0.26.1-rc.0, not v0.26.0?

Reply:
Manually updated it, somehow the go-get-k8s.sh picked this one up